src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -141,10 +179,10 @@ public class CoinstakeContext
         /// multiplied by this value. See <see cref="GetSplitStake(int)"/>.</summary>
```

Comment:
missing opening summary tag and also just before that there are two consts with old comments, xmldoc them pls